sdk/tables/azure-data-tables/test/ut/table_client_test.cpp

```diff
@@ -238,15 +238,17 @@ namespace Azure { namespace Data { namespace Test {
     EXPECT_FALSE(response.Value.ETag.empty());
 
     entity.Properties["Product"] = TableEntityProperty("Tables2");
-    auto updateResponse = m_tableClient->UpdateEntity(entity);
+    auto updateResponse
+        = m_tableClient->UpdateEntity(entity, Azure::Data::Tables::Models::UpdateMode::Update);
```

Comment:
Do the tests here validate the difference between Update and Merge, based on the observable behavioral difference on the returned response or model?

E.g. if I change Update to Merge here, will the test fail or continue to pass?

It would be a good idea to add some assert to check the enum is honored as expected.